### ISSUE TITLE
how can I create and use custom Context?

### ISSUE THREAD

**Issue by naftali100** [badfarm/zanzara#42]
if there is not option like this please add

**Comment by cheeghi** [comment]
What do you mean by "custom context"?

**Comment by naftali100** [comment]
an option to add some functions to the Context
like sendMessage that can take any type of variable and convert to text
or custom getters like getEffectedFile

**Comment by cheeghi** [comment]
Right now there isn't a way to do it, but it may be useful actually, I keep this issue open

**Comment by cheeghi** [comment]
Available in version `0.7.0`:

```php
<?php

class CustomContext extends \Zanzara\Context
{

    public function customMethod()
    {
        $message = $this->update->getMessage();
        $this->sendMessage('Hello ' . $message->getFrom()->getFirstName());
    }

}

$config = new Config();
$config->setContextClass(CustomContext::class);
$bot = new Zanzara("token", $config);

$bot->onUpdate(function (CustomContext $ctx) {

});

$bot->run();

```

**Comment by naftali100** [comment]
Thank you!
